---
layout: page
title: Applications
permalink: /applications/
---

<h2>Environmental DNA Applications & State of the Art</h2>

<strong>Environmental DNA (eDNA)</strong> can be broadly defined as organismal community DNA extracted from bulk samples (not from individual specimens) such as soil, water, or air.  The DNA may come from the organisms themselves (bacteria, fungi, diatoms) or from shed cells or reproductive propagules from larger organisms (dead, dormant, or alive).  DNA may also come from mixedmixed assemblages (ex. Malaise traps, kicknet samples). There are several methods that can be used to detect and/or quantify individual species or whole communities from eDNA.

Species detection using <strong>reverse transcription PCR (RT-PCR)</strong> uses species-specific primers so that organism-specific eDNA can be detected and quantified. RT-PCR is sometimes also referred to as quantitative PCR (qPCR) and a newer technique is now available called digitial droplet PCR (ddPCR) where many individual reactions can be carried out in parallel in their own microscopic micellar droplets.

<strong>DNA metabarcoding</strong> is a technique where broad range primers and high throughput sequencing is carried out, such as on the popular Illumina MiSeq platform, to generate very large community datasets.  

<strong>Metagenomics</strong> is a primer-free technique, sometimes referred to as "shotgun" sequencing where random fragments of eDNA are sequenced and comapred with reference sequence databases.  Related techniques include genome skimming where shallow sequencing of samples detect the most abundant parts of the genome such as highly repetitive ribosomal or mitochondrial sequences and mitogenome sequencing.

<h3 class="text-info"><a id="genomicmethoddevelopment">Genomic Method Development</a></h3>

Novel ways to sample eDNA in the field are now readily available, with kits available to sample a variety of substrates (feces, soil, water, plant/animal tissue), and new methods such as passive sampling being developed.  As reference databases grow, new primers and protocols are continually being developed to improve specificity (target taxa) or breadth ('universal' primers).  Automation is increasingly being used to improve throughput.  Companies that provide molecular biology and sequencing solutions are becoming increasingly available.  This is a very rapidly chaning field, with new sequencing platforms available to produce very long sequences (PacBio), massive amounts of shorter sequences (ex. Illumina NovaSeq), and provide portability (Oxford).  Some of these platforms provide strategies to help enrich rare targets or exclude host sequences (Nanopore adaptive sampling).  

<h3 class="text-info"><a id="bioinformaticsmethoddevelopment">Bioinformatics Method Development</a></h3>

Reproducibility and sharability of the tools used to process DNA metabarcode and metagenomic datasets can be facilitated using versioned bioinformatic pipelines.  Quality control of sequence reads, reduction of noise due to technical artefacts, and taxonomic assignment are controlled by applying the latest bioinformatic tools. This is an area of rapid development in terms of the sequencing platforms/chemistry and new algorithms used for clustering, identifying important features in the data (keystone species, bioindicator species, etc.), and for taxonomic assignment.  Reference sequence databases, local to globally representative sets, substrate-specific sets, their completeness and accuracy, affect the reliability of the resulting taxonomic assignments.

<h3 class="text-info"><a id="invasivespecies">Invasive Species</a></h3>

RT-PCR is the main tool used to detect and quantify species of interest from eDNA samples. Because this technique is so sensitive, it is important to use a validated assay with appropriate controls.  Interpretation of results will depend on factors such as the limits of detection of the assay and impact of inhibition.  The use of DNA metabarcoding to broadly survey for invasive species may be limited by incomplete reference sequence databases and low starting template amounts (false negatives).

<h3 class="text-info"><a id="freshwaterbiomonitoring">Freshwater Biomonitoring</a></h3>

Due to the scalability, throughput, and cost-effectiveness of eDNA tools, they have been an attractive choice to use for biomonitoring efforts.  Both RT-PCR and DNA metabarcoding methods have been used to monitor target taxa (ex. invasive species), bioindicator assemblages (ex. Ephemeroptera, Plectoptera, Trichoptera, etc.), and whole communities.  Environmental DNA from water, sediment, and stream-bottom macroinvertebrates can be targeted on their own or as part of more comprehensive surveys.  Some studies have begun to use multi-marker metabarcoding to target major components of the biological community such as bacteria, fungi, diatoms, macroinvertebrates, so as to assess trophic interactions, community stability, and identify keystone species.  Environmental DNA tools have been benchmarked against conventional methods in numerous studies and have started to become incorporated into governmental monitoring programs in Europe and Canada.

<h3 class="text-info"><a id="marinebiomonitoring">Marine Biomonitoring</a></h3>

The use of DNA-based methods to detect organisms in sea water has a long history of use.  Early tools included DNA-fingerprinting and microarrays.  Today, innovative sampling methods (ex. automated collectors) and eDNA molecular biology methods (ex. modular, nanopore based sequencing setups that provide quick processing time and near real-time results) have been used to survey the breadth of marine life from bacteria to whales.  In the US, great strides have already been made to bring together research scientists, government agencies, and environmental managers to co-develop and apply eDNA science solutions.

<h3 class="text-info"><a id="endangeredspecies">Endangered Species</a></h3>

Similar to studies that use eDNA tools to detect invasive species, RT-PCR and DNA metabarcoding can be used to screen for endangered species or may be identified from broader community surveys.  False negatives due to lack of reference database representation and low detection probability are concerns.  

<h3 class="text-info"><a id="ecologicalstudies">Ecological Studies (foodwebs, diet, trophic interactions)</a></h3>

Studies may use single or multi-marker DNA metabarcoding methods to survey the breadth of target communities.  eDNA tools are ideal for studying plant-insect interactions, for example, from the trace DNA left by pollinators on plant surfaces.  For studies that incorporate more diverse taxa (microbes to macrobes) it can be challenging to use automated tools to map functional information (ex. resource-consumer relationships, feeding strategy, ecological guild, etc.) to all taxa due to database incompleteness.  This is a very active area of research and both taxonomic and functional databases continue to grow.

<h3 class="text-info"><a id="remediationandrestorationmonitoring">Remediation and Restoration Monitoring</a></h3>

Due to the non-invasive nature of many eDNA based tools, both RT-PCR to assay bioindicator species and/or whole community surveys using DNA metabarcoding are widely used.  eDNA-based tools are often combined with other monitoring tools such as conventional biological surveys, water/soil physical-chemical measures, incorporation of landscape variables, etc.

<h3 class="text-info"><a id="fisheriesandaquaculture">Fisheries and Aquaculture</a></h3>

Numerous eDNA based tools exist for monitoring target species (ex. validated RT-PCR assays) and whole communities (DNA metabarcoding).  Much effort has gone into validating the use sequence read data to estimate the abundance or relative abundance of target fish.  Likewise, much effort has been put into reference sequence databases (ex. whole mitochondrial genome sequences) to facilitate multi-marker fish taxonomic assignments.

